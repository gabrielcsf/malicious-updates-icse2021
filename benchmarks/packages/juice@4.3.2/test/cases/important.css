a {
  color: red !important;
  background: blue !important;
  border-style: dashed !important;
}

#a {
  color: purple;
  background: purple;
  border-style: solid;
}
