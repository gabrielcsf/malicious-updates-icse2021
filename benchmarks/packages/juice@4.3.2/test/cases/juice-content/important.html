<html>
<head>
    <style>
        p {
            color: red !important;
        }
        span {
            color: blue;
        }
    </style>
</head>
<body>
    <p>hi</p>
    <span>there</span>
    <p style="color: blue !important">again</p>
</body>
</html>
