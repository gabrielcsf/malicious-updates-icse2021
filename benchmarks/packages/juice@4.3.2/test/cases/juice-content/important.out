<html>
<head>
    
</head>
<body>
    <p style="color: red !important;">hi</p>
    <span style="color: blue;">there</span>
    <p style="color: blue !important;">again</p>
</body>
</html>
